### PR TITLE
chore: fix typos in various places

### PR DESCRIPTION
**Pull request by HigherOrderLogic**

### CODE REVIEW

File: docs/docs/config/bluetooth.md
Comment (caksoylar) [inline docs/docs/config/bluetooth.md:14]: Unfortunately this requires re-aligning the table, i.e. extending the width of this column by one in all other rows as well: https://github.com/zmkfirmware/zmk/actions/runs/8925194729/job/24520874985?pr=2290#step:4:137

**Review by caksoylar — APPROVED** [review]
Thank you, docs changes (and others, too) LGTM. All other changes are in Kconfig/yml descriptions or code comments, so should be safe to merge, but I'll leave it to the core maintainers.

**Review by petejohanson — APPROVED** [review]
All good catches. Thanks!

**Pull request merged**
